src/js/components/header.js

```diff
@@ -45,7 +41,9 @@ let HeaderLogin = React.createClass({
       </div>
     }
   }
-});
-
-
-export default Header;
+}
+HeaderLogin.propTypes = {
```

Comment:
Nope. http://www.newmediacampaigns.com/blog/refactoring-react-components-to-es6-classes